Fix incorrect register value for ProcessNameFormat (#22535)
dotnet/docs: docs/framework/debug-trace-profile/performance-counters-and-in-process-side-by-side-applications.md

 [!code-csharp[Conceptual.PerfCounters.InProSxS#1](../../../samples/snippets/csharp/VS_Snippets_CLR/conceptual.perfcounters.inprosxs/cs/regsetting1.cs#1)]
 [!code-vb[Conceptual.PerfCounters.InProSxS#1](../../../samples/snippets/visualbasic/VS_Snippets_CLR/conceptual.perfcounters.inprosxs/vb/regsetting1.vb#1)]  
  

>  When you make this registry change, Perfmon.exe displays the names of applications that target the .NET Framework 4 as *application*_`p`*processID*, where *application* is the name of the application, and *processID* is the application's process identifier. For example, if an application named myapp.exe loads two instances of the common language runtime, Perfmon.exe may identify one instance as myapp_1416 and the second as myapp_3160. The runtime identifier only disambiguates the runtimes within a process; it does not provide any other information about the runtime. (For example, the runtime ID has no relation to the version or the SKU of the runtime.)  

Reviewer: ```suggestion
 When you make this registry change, and if .NET Framework 4 or later is installed, Perfmon.exe displays the names of applications as *application*_`p`*processID*, where *application* is the name of the application, and *processID* is the application's process identifier. For example, if an application named myapp.exe loads two instances of the common language runtime, Perfmon.exe may identify one instance as myapp_1416 and the second as myapp_3160.
```

>  If the .NET Framework 4 or a later version is installed, the registry change also affects applications that were developed using earlier versions of the .NET Framework. These appear in Perfmon.exe as *application_*`p`*processID*, where *application* is the application name and *processID* is the process identifier. For example, if the performance counters of two applications named myapp.exe are monitored, one might appear as myapp_23900 and the other as myapp_24908.

Reviewer: ```suggestion
```

This seems to be identical to apps that target .NET Framework 4+.
